pkgs/development/r-modules/generate-r-packages.R

```diff
@@ -12,13 +11,11 @@ if ("release" %in% biocVersion$BiocStatus) {
 } else {
   biocVersion <-  max(as.numeric(as.character(biocVersion$Bioc)))
 }
-dates <- stream_in(url("https://packagemanager.rstudio.com/__api__/repos/2/transaction-dates"), verbose = FALSE)
-snapshotDate <- as.Date(dates[nrow(dates), "alias"])
 
 mirrorUrls <- list( bioc=paste0("http://bioconductor.statistik.tu-dortmund.de/packages/", biocVersion, "/bioc/src/contrib/")
                   , "bioc-annotation"=paste0("http://bioconductor.statistik.tu-dortmund.de/packages/", biocVersion, "/data/annotation/src/contrib/")
                   , "bioc-experiment"=paste0("http://bioconductor.statistik.tu-dortmund.de/packages/", biocVersion, "/data/experiment/src/contrib/")
-                  , cran=paste0("https://packagemanager.rstudio.com/cran/", snapshotDate, "/src/contrib/")
+                  , cran=paste0("https://cran.r-project.org/src/contrib/")
```

Comment:
Again, an argument or parameter would be nice to DIY mirrors.

Reply:
Out of scope but yes it would be nice to make it more configurable and maybe reduce duplication using the known fetchurl mirrors.